src/VisualStudio/Core/Def/Remote/VisualStudioRemoteHostClientProvider.cs

```diff
@@ -44,6 +48,7 @@ internal sealed class Factory : IWorkspaceServiceFactory
         public Factory(
             VisualStudioWorkspace vsWorkspace,
             IVsService<SVsBrokeredServiceContainer, IBrokeredServiceContainer> brokeredServiceContainer,
+            SVsServiceProvider serviceProvider,
```

Comment:
would it make sense to take an IVsService like above, since you just use the iServiceProvider side of htis?